client_cli/src/main.rs

```diff
@@ -76,6 +78,12 @@ pub struct Args {
     /// Sets a config file path
     #[structopt(short, long)]
     config: Option<Configuration>,
+    /// Output results in machine-readable JSON
+    #[structopt(short, long)]
+    json: bool,
```

Comment:
I believe this flag should be enabled by default because JSON output is more useful in most cases:
+ pipe this CLI tool and parse it as JSON
+ easy to copy and paste into JSON explorers

Reply:
If we agree JSON is more useful even for non-script usage we could then drop RON alltogether and make it JSON-only. @appetrosyan @Erigara what do you think?

Reply:
@QuentinI I'm for leaving only `json`. If we really need to we can always expand range of supported formats, but do other way around may cause some confusion for the users.

Reply:
For now, my opinion is that we should use JSON-only. If we decide to change it to something else later, there should be a good justification. 

It makes some sense to change the config to TOML, but only the config, the rest is better served with a simple format such as JSON.